README.md

```diff
@@ -65,13 +66,24 @@ Note that using `run.sh` sets these temporarily in the shell.
     
 ### Run OpenDroneMap
 
-First you need a set of images, taken from a drone or otherwise. Example data can be cloned from https://github.com/OpenDroneMap/odm_data
+First you need a set of images, taken from a drone or otherwise. Example data can be obtained from https://github.com/OpenDroneMap/odm_data
+
+Next, you need to copy over the settings file `default.settings.yaml` and edit it. The only setting you must edit is the `project-path` key. Set this to an empty directory within projects will be saved. There are many options for tuning your project. See the [wiki](https://github.com/OpenDroneMap/OpenDroneMap/wiki/Run-Time-Parameters) or run `python run.py -h`
```

Comment:
I'm a little confused here, I take `default.settings.yaml` and copy it over (where?).

I think I see where this ties in with the new `--name` parameter, but I'm a little confused. I think other people will be as well. I wonder whether this ability to do project management and organization via command line should be coded in a module separate from `run.py`. Borrowing from Unix philosophy, make lot of small utilities, that work well separately, but flexible enough to be combined.